test/modules/standard/jama/TestMatrix.chpl

```diff
@@ -0,0 +1,403 @@
+use Jama;
+use Math;
+
+/** TestMatrix tests the functionality of the Jama Matrix class and associated decompositions.
+<P>
+Run the test from the command line using
+<BLOCKQUOTE><PRE><CODE>
```

Comment:
@Kyle-B for whole array assignment is the preference to do 'p(..) = piv(..)' or just a simple 'p = piv'?(Jama.chpl:1612)